src/collection/components/fragment/FragmentDetail.tsx

```diff
@@ -81,6 +81,8 @@ const FragmentDetail: FunctionComponent<FragmentDetailProps> = ({
 		<BlockIntro
 			content={collectionFragment.custom_description || ''}
 			title={collectionFragment.custom_title || ''}
+			headingType="h3"
```

Comment:
@bertyhell Enig idee waarom deze niet doorgevoerd wordt? 🤔 Als ik de inhoud van BlockIntro hier rechtstreeks plaats, werkt het wel. 😕

Reply:
ik begrijp je comment niet. Kan je hier zetten wat je wil dat er gebeurt en wat er nu gebeurt?

Reply:
@bertyhell 
Ik heb BlockIntro aangepast om de meegegeven heading type te gebruiken in de titel in dit blok. Nu hier wordt deze niet toegepast en ik zie niet in waarom. In comp lib stories, lukt dit perfect, en als ik de inhoud van BlokIntro hier plaats ipv het blok, lukt dit ook. Maar er komt een h1 titel, ook al geef ik h3 mee. 😅

Reply:
wss een npm caching issue want bij mij is het een h3:
![image](https://user-images.githubusercontent.com/1710840/82678839-9b957e00-9c4a-11ea-8dd0-46df2eba60ae.png)

Reply:
Ik heb het over het Intro-blok, dus een text-fragment. Dit is een media-fragment. Bij een text-fragment is het bij mij h1.

Reply:
bij mij is het voor een text fragment een h3:
![image](https://user-images.githubusercontent.com/1710840/82788066-08d22a80-9e68-11ea-84e6-8ec3518e6826.png)